openlibrary/plugins/openlibrary/js/type_changer.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Functionality for TypeChanger.html
+ */
+
+export function initTypeChanger() {
```

Comment:
```suggestion
export function initTypeChanger(elem) {
```
Can you pass a reference to the type changer element to this function?